packages/neos-ui/src/index.js

```diff
@@ -129,6 +130,7 @@ function * application() {
     const menu = yield system.getMenu;
     const configuration = yield system.getConfiguration;
     const translations = yield system.getTranslations;
+    tmp.translations = translations;
```

Comment:
!!! REALLY UGLY HACK!! I did not see another way to receive the translations except through an ugly side-effect.